fedimint-bitcoind/src/bitcoincore_rpc.rs

```diff
@@ -314,27 +207,26 @@ impl IBitcoindRpc for ElectrumClient {
             bail!("Electrum backend does not contain the block at {height}H yet");
         }
 
-        fedimint_core::task::block_in_place(|| {
-            let txid = transaction.txid();
-
-            let output = transaction
-                .output
-                .first()
-                .ok_or(format_err!("Transaction must contain at least one output"))?;
-
-            // if transaction is confirmed, we're going to find the confirmation event in
-            // the history of ifs first output
-            Ok(self
-                .0
-                .script_get_history(&output.script_pubkey)?
-                .iter()
-                .any(|history_item| {
-                    (history_item.height as u64) == height && history_item.tx_hash == txid
-                }))
-        })
```

Comment:
Removing `block_in_place` here could lead to starving other tasks for no good reason since `script_get_history` might block.